jquery.xblock.js

```diff
@@ -154,6 +154,15 @@
                     return (lmsBaseURL + '/courses/' + courseId + '/xblock/' + usageId +
                             '/handler/' + handlerName);
                 },
+                local_overrides: {
+                    discussion: function(element, DiscussionUtil) {
+                        var courseId = $(element).data('course-id');
+
+                        DiscussionUtil.localUrls.push('user_profile');
+                        DiscussionUtil.force_async = true;
+                        DiscussionUtil.route_prefix = 'courses/' + courseId + '/discussion'
+                    }
+                }
```

Comment:
@antoviaque sure I can remove it from here, but this is required to keep discussion working with Apros, so we'll need some way to specify this client-specific overrides.